src/main/java/com/redhat/cloud/common/clowder/configsource/ClowderConfigSource.java

```diff
@@ -224,6 +224,13 @@ public String getValue(String configKey) {
                         String url = existingValues.get(QUARKUS_DATASOURCE_JDBC_URL).getValue();
                         if (url.contains(":tracing:")) {
                             tracing = "tracing:";
```

Comment:
That should not longer be needed, as OpenTracing is deprecated and probably even no longer supported by Quarkus

Reply:
Do you think we can remove it already? Although unlikely, it could be used by a project that depends on clowder-quarkus-config-source.